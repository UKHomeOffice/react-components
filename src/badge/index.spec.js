import React from 'react';
import { render } from 'enzyme'
import Badge from 'govuk-react-components/badge';

describe('The Badge Component', () => {
  it('should render children that are passed in', () => {
    expect(
      render(<Badge>Sample Text</Badge>)
    ).toMatchSnapshot()
  });

  it('should append classNames when attribute is passed in', () => {
    expect(
      render(<Badge className="my-additional-classname">Sample Text</Badge>)
    ).toMatchSnapshot();
  });

});