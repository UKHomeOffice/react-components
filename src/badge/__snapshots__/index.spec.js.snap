// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Badge Component should append classNames when attribute is passed in 1`] = `
<strong
  class="phase-tag my-additional-classname"
>
  Sample Text
</strong>
`;

exports[`The Badge Component should render children that are passed in 1`] = `
<strong
  class="phase-tag"
>
  Sample Text
</strong>
`;
